Example/RSSwiftNetworking.xcodeproj/project.pbxproj

```diff
@@ -357,6 +357,7 @@
 				ONLY_ACTIVE_ARCH = YES;
 				SDKROOT = iphoneos;
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
+				SWIFT_VERSION = 5.0;
```

Comment:
This value is set from a predefined list in the build settings. It is not recommended/good to manually set the value by editing the `pbxproj` file, since the build system will validate it against a hardcoded list depending on the Xcode version.